charts/chart.line.js

```diff
@@ -128,6 +128,10 @@ line.prototype.getSpec = function() {
 
 
 function getLineMark(config, metadata){
+    interpolation = "linear";
```

Comment:
@keizer619 "linear" is the default option. If the users want they can use any of the values out of the following
 - linear
 - step-before
 - step-after 
 - basis 
 - basis-open 
 - basis-closed 
 - bundle 
 - cardinal 
 - cardinal-open 
 - cardinal-closed 
 - monotone